src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -262,6 +266,11 @@ public void initialize() {
     chatUserListView.setCellFactory(param -> new ChatUserListCell(uiService));
 
     autoCompletionHelper.bindTo(messageTextField());
+
+    splitPane.widthProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
EHMM GEO????

Reply:
What does this fix I am certain there is a clean way for this

Reply:
It fixes the user list to a fixed width instead of specifying that as a percentage of the monitor size. And yes, I have searched thoroughly, there is no other way to do this.

Reply:
ok fine